src/test01/app.html

```diff
@@ -1,3 +1,39 @@
 <div class="container">
-Your code here, see desktop/tablet/mobile.png.
-</div>
\ No newline at end of file
+
+	<div class="row extra-bottom-padding-small">
+		<h2 class="text-center">Book an Appointment</h2>
+		<h5 class="text-center">Our Investing Topics</h5>
+	</div>
+
+	<div class="row extra-bottom-padding-medium">
+		<h4 class="text-center">Pick a topic, then we'll find a time and place</h4>
+	</div>
+
+	<div ng-if="buttons.length!=5 && buttons.length!=1" class="row extra-bottom-padding-medium" ng-all-same-height target-element="button">
```

Comment:
I decided to use the ng-if directive here to ensure that the elements were displayed appropriately depending on the cases provided in the readme. 

Notice that the ng-all-same-height directive is initialized along with the target-element being set to "button" to ensure that all button child elements are set to an equal height.